.github/workflows/gen.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Run Node Client Generator and Create Pull Request
+on:
+  schedule:
+    # Run at the end of every day
+    - cron: 0 0 * * *
+jobs:
+  create_pr:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout this repo
+      uses: actions/checkout@v2
+      with:
+        repository: googleapis/google-cloudevents-nodejs
+        path: google-cloudevents-nodejs
+    - name: Checkout client repo
+      uses: actions/checkout@v2
+      with:
+        repository: googleapis/google-cloudevents
+        path: google-cloudevents
```

Comment:
Why does this repo need to be checked out? This workflow doesn't seem to use it at all and I can't find in unmarshal-autogen the reasoning either. If you do need it, we need better comments to improve the maintainability of this workflow.

Reply:
The Google CloudEvents repo contains the protos and more importantly, JSON schemas for all of the events. We need those schemas to generate our types.

I can add a comment.